### DISCUSSION TITLE
Sponsors allowed for  Armenia but seems impossible to apply

### DISCUSSION THREAD

**Discussion by boozook** [community/community#69263] [Other Feature Feedback, Questions, & Ideas]
From today GitHub Sponsors is available for me 🎉 **in the Armenia**, that's important.

But latest step before submitting application is to connect my Stripe account. But Stripe doesn't have Armenia in list of supported countries, so I can't create Stripe account.

_Actually I able to register new account in Stripe, but definitely can't provide my requisites, billing, etc.. to complete Stripe's KYC verification._

Please provide some hint for me please.

**Comment by elringus** [comment]
@boozook Hey, Sorry for the ping, but I'm curious if you were you able to activate sponsors from Armenia?

**Reply by boozook** [reply]
That’s okay, don’t worry 😉 
[Yup, I did it](https://github.com/sponsors/boozook?o=sd&sc=t), thanks.
